src/History/components/ProgressBar.tsx

```diff
@@ -0,0 +1,86 @@
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+interface ProgressBarProps {
+  currentProgress: number;
+  maximumProgress: number;
+}
+
+// history_progressbar 현재 진행 마크용
+interface MarkerProps {
+  value: number;
+}
+
+const ProgressBar = ({ currentProgress, maximumProgress }: ProgressBarProps) => {
+  const leftValue = (currentProgress / maximumProgress) * 20.0 - 0.6;
+  return (
+    <div css={progressBarContainer}>
+      <StProgressBarWrapper>
+        <progress value={currentProgress} max={maximumProgress}></progress>
```

Comment:
안에 요소가 없는 태그는 이렇게 써줍시다
```suggestion
        <progress value={currentProgress} max={maximumProgress}/>
```

Reply:
넹~!